Sources/KituraContracts/CodableQuery/QueryDecoder.swift

```diff
@@ -60,18 +60,24 @@ public class QueryDecoder: Coder, Decoder, BodyDecoder {
      */
     public var dictionary: [String : String]
 
-    
+    /**
+     A `JSONDecoder.DateDecodingStrategy` date decoder used to determine what strategy
+     to use when decoding the specific date.
+     */
+    public var dateDecoder: JSONDecoder.DateDecodingStrategy
     /**
      Initializer with an empty dictionary for decoding from Data.
      */
-    public override init() {
+    public init(dateDecodingStrategy: JSONDecoder.DateDecodingStrategy = .formatted(Coder().dateFormatter)) {
+        self.dateDecoder = dateDecodingStrategy
```

Comment:
This is inconsistent with QueryEncoder and with the precedent set by other Decoders.  I'd leave the initializer signature alone, and make this
```swift
self.dateDecoder = .formatted(Coder().dateFormatter)
```
as you have with QueryEncoder.  The user can then choose to set `dateDecoder` to a custom value after initialization.